src/style.css

```diff
@@ -2,2 +1,82 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  /* Space between logo and nav */
+  align-items: center;
+  /* Vertically center the content */
+  padding: 1rem;
+  background-color: #fffff;
+  color: white;
+  width: 100%;
+}
+
+.logo {
+  margin-left: 0;
+  /* No margin to ensure it's at the left edge */
+}
+
+.logo a {
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  /* Transform logo text to uppercase */
+}
+
+.logo img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  margin-left: auto;
+  /* Pushes the nav to the far right */
+}
+
+.nav ul {
+  list-style: none;
```

Comment:
The styling for 'ul' elements should be done using classes instead of tag names to avoid styling unwanted elements and to maintain specificity. Consider using a class like '.nav__list' for styling.